src/uct/base/uct_iface.h

```diff
@@ -90,6 +90,16 @@ enum {
 
 
 /**
+ * This macro should be deleted after UCT gather/scatter IOV interface changed
+ */
+#define UCT_CHECK_PARAM_IOV \
```

Comment:
let's pass the variable names to the macro as parameters (_iov, _iovcnt, _buffer, _length, _memh)